src/components/pages/AdminHome/AdminHomeContent.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CourseCard from './CourseCard';
+import ApplicationCard from './ApplicationCard';
+import { Layout, Card } from 'antd';
+import '../../../styles/AdminDashboardHome/index.less';
```

Comment:
Can we remove `Card` if it not being used

Reply:
Done